src/ol/interaction/DragBox.js

```diff
@@ -269,6 +277,26 @@ class DragBox extends PointerInteraction {
    * @param {import("../MapBrowserEvent.js").default} event Event.
    */
   onBoxEnd(event) {}
+
+  /**
+   * Activate or deactivate the interaction.
+   * @param {boolean} active Active.
+   * @observable
+   * @api
+   */
+  setActive(active) {
+    if (!active) {
+      this.box_.disposeInternal();
```

Comment:
```suggestion
      this.box_setMap(null);
```
`disposeInternal` is called by `dispose` to permanently destroy an object. While it would do the job here it makes more sense to me to call `this.setMap(null)` directly here.

Reply:
I see.. I missed what `dispose` strictly means, and I agree with calling `this.setMap(null)` instead.

https://github.com/openlayers/openlayers/pull/15671/commits/bcd923f7fa62f3b9eb3ff2a341078036b857ff7b includes this suggestion. Thanks!